anchor/common/qbft/src/tests.rs

```diff
@@ -87,31 +91,52 @@ fn construct_and_run_committee<D: Data + Default + 'static>(
             move |message| msg_queue.borrow_mut().push_back((id, message)),
         );
         instances.insert(id, instance);
+        active_instances.insert(id);
     }
 
     TestQBFTCommittee {
         msg_queue,
         instances,
+        active_instances,
     }
 }
 
 impl<D: Default + Data, S: FnMut(Message<D>)> TestQBFTCommittee<D, S> {
-    fn wait_until_end(mut self) {
+    fn wait_until_end(mut self) -> i32 {
```

Comment:
What about a more descriptive name?

```suggestion
    fn process_msgs_and_count_consensus(mut self) -> i32 {
```